examples/deployment/kubernetes/trillian-log-deployment.yaml

```diff
@@ -66,7 +66,8 @@ spec:
           - --source=logserver:http://localhost:8091/metrics
           - --pod-id=$(POD_NAME)
           - --namespace-id=$(POD_NAMESPACE)
-          - --metrics-resolution=60s
+          - --scrape-interval=5s
```

Comment:
Fairly idle question - are these related to the prom2sd upgrade or the cost of metrics you mentioned yesterday.

Reply:
prom2sd upgrade - the flags have changed...